ntbs-service/DataMigration/MigrationRepository.cs

```diff
@@ -199,6 +236,52 @@ public async Task<IEnumerable<dynamic>> GetTransferEvents(List<string> legacyIds
             }
         }
 
+        public async Task<IEnumerable<dynamic>> GetOutcomeEvents(List<string> legacyIds)
+        {
+            using (var connection = new SqlConnection(connectionString))
+            {
+                connection.Open();
+                return await connection.QueryAsync(OutcomeEventsQuery, new {Ids = legacyIds});
+            }
+        }
+
+        public async Task<IEnumerable<dynamic>> GetMigrationMBovisAnimalExposure(List<string> legacyIds)

```

Comment:
Can we commonise this mbovis methods?

`public async Task<IEnumerable<dynamic>> GetMigrationMBovisAnimalExposure(List<string> legacyIds) 
=> GetMigrationMBovis(MigrationMBovisAnimalExposureQuery, legacyIds)`

Reply:
I think you're right - and there's a bunch more methods above that fit the same format!